Felienne/hedy
# [CHORE] Removes mandatory account mail address

**Description**
In this PR we remove the mail address as mandatory when creating a new account. We received feedback multiple times that students don't have any mail address or don't have access to it at the moment of account creation. As the access is not essential for the creation and using of the account, it is for account validation and password resetting. We start this big chore by first removing all dependencies on a mail address, next we should look at the replacement for accounts without a mail address to still be able to reset their account. It might be best to solve this in a separate issue as the task will grow a bit too big for one PR to be tested otherwise.

**Fixes**
This PR fixes #2075.

**How to test**
Create a new account and verify that you are able to do this without entering a mail account. When entering a mail account it is still validated if the address is valid and doesn't already exist. Also verify that the multiple account creation for teacher still works.
